<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1492</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="videoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>721</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="currenttime">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 138, 133);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="progress">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>450</y>
      <width>590</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="totaltime">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>440</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 138, 133);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="volume">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>40</y>
      <width>22</width>
      <height>350</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="volumeLabel">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>450</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 138, 133);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="speedComboBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>520</y>
      <width>173</width>
      <height>34</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Pausebutton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>520</y>
      <width>91</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Playbutton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Filebutton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>520</y>
      <width>101</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1492</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
